terraform.tfvars.example

```diff
@@ -61,8 +61,11 @@ gpuNodeSize = [
   "g5.2xlarge"
 ]
 
-# The NVIDIA driver version for GPU node group.
-gpuNvidiaDriverVersion = "535.54.03"
+# Input configuration for the GPU operator chart deployed with helm release.
+# By setting key 'enable' to 'true', GPU operator will be deployed.
+# 'helm_repository' is an URL for the repository of the GPU operator helm chart, where 'helm_version' is its respective version of a chart.
+# 'chart_values' is used for changing default values.yaml of the GPU operator chart.
+gpu_operator_config = 
```

Comment:
Is this typo or gpu_operator_config should be like empty dict?

Reply:
This was created by github-actions. I'll add a real example